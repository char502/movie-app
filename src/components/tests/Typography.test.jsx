import React from "react";
import {shallow} from 'enzyme';
import { 
    Title, MediaListTitle, SearchResultTitle, 
    Blurb, Caption, SearchPageBlurb, MediaDetail,
    Overview
} from '../Typography';
import toJson from 'enzyme-to-json';

it('Button Snapshots', () => {
    expect(toJson(shallow(<Title />))).toMatchSnapshot();
    expect(toJson(shallow(<MediaListTitle />))).toMatchSnapshot();
    expect(toJson(shallow(<SearchResultTitle />))).toMatchSnapshot();
    expect(toJson(shallow(<Blurb />))).toMatchSnapshot();
    expect(toJson(shallow(<Caption />))).toMatchSnapshot();
    expect(toJson(shallow(<SearchPageBlurb />))).toMatchSnapshot();
    expect(toJson(shallow(<MediaDetail />))).toMatchSnapshot();
    expect(toJson(shallow(<Overview />))).toMatchSnapshot();
});