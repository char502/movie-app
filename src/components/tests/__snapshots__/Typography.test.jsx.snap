// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Snapshots 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  text-align: center;
  text-transform: uppercase;
  font-family: Arial;
  font-size: ",
          [Function],
          ";
  font-weight: 900;
  letter-spacing: -2.5px;
  background: ",
          [Function],
          ";
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  margin: 0;
",
        ],
      },
      "displayName": "styled.h1",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h1",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 2`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bwzfXH",
        "isStatic": false,
        "rules": Array [
          "
  text-align: center;
  text-transform: uppercase;
  font-family: Arial;
  font-size: ",
          [Function],
          ";
  font-weight: 900;
  letter-spacing: -2.5px;
  background: ",
          [Function],
          ";
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  margin: 0;
",
          "
  text-align: left;
  padding-top: ",
          [Function],
          ";
",
        ],
      },
      "displayName": "Styled(styled.h1)",
      "foldedComponentIds": Array [
        "sc-bdVaJa",
      ],
      "render": [Function],
      "styledComponentId": "sc-bwzfXH",
      "target": "h1",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 3`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-htpNat",
        "isStatic": false,
        "rules": Array [
          "
  text-align: center;
  text-transform: uppercase;
  font-family: Arial;
  font-size: ",
          [Function],
          ";
  font-weight: 900;
  letter-spacing: -2.5px;
  background: ",
          [Function],
          ";
  -webkit-background-clip: text;
  -webkit-text-fill-color: transparent;
  margin: 0;
",
          "
  text-align: left;
  position: relative;
  font-size: ",
          [Function],
          ";
  padding-bottom: ",
          [Function],
          ";
",
        ],
      },
      "displayName": "Styled(styled.h1)",
      "foldedComponentIds": Array [
        "sc-bdVaJa",
      ],
      "render": [Function],
      "styledComponentId": "sc-htpNat",
      "target": "h1",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 4`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bxivhb",
        "isStatic": false,
        "rules": Array [
          "
  font-family: Helvetica;
  font-weight: 700;
  color: ",
          [Function],
          ";
",
        ],
      },
      "displayName": "styled.div",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bxivhb",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 5`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-ifAKCX",
        "isStatic": false,
        "rules": Array [
          "
  font-family: Helvetica;
  font-weight: 700;
  color: ",
          [Function],
          ";
",
          "
  position: absolute;
  bottom: 0;
  left: 0;
  padding-left: ",
          [Function],
          ";
  font-size: ",
          [Function],
          ";
  opacity: 0;
  transition: 0.2s ease-in-out;
",
        ],
      },
      "displayName": "Styled(styled.div)",
      "foldedComponentIds": Array [
        "sc-bxivhb",
      ],
      "render": [Function],
      "styledComponentId": "sc-ifAKCX",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 6`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-EHOje",
        "isStatic": false,
        "rules": Array [
          "
  font-family: Helvetica;
  font-weight: 700;
  color: ",
          [Function],
          ";
",
          "
  width: 60%;
  padding: ",
          [Function],
          ";
",
        ],
      },
      "displayName": "Styled(styled.div)",
      "foldedComponentIds": Array [
        "sc-bxivhb",
      ],
      "render": [Function],
      "styledComponentId": "sc-EHOje",
      "target": "div",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 7`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bZQynM",
        "isStatic": false,
        "rules": Array [
          "
  font-family: Helvetica;
  font-size: ",
          [Function],
          ";
  padding-bottom: ",
          [Function],
          ";
  color: ",
          [Function],
          ";
",
        ],
      },
      "displayName": "styled.h3",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bZQynM",
      "target": "h3",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;

exports[`Button Snapshots 8`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-gzVnrw",
        "isStatic": false,
        "rules": Array [
          "
  font-family: Helvetica;
  font-size: ",
          [Function],
          ";
  padding-bottom: ",
          [Function],
          ";
  color: ",
          [Function],
          ";
",
          "
  font-size: ",
          [Function],
          ";
",
        ],
      },
      "displayName": "Styled(styled.h3)",
      "foldedComponentIds": Array [
        "sc-bZQynM",
      ],
      "render": [Function],
      "styledComponentId": "sc-gzVnrw",
      "target": "h3",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
